modules/tournament/src/main/TournamentScheduler.scala

```diff
@@ -404,41 +404,38 @@ Thank you all, you rock!"""
           // Because berserking lowers the player rating
           _ map { _.copy(noBerserk = true) }
         },
-      // hourly crazyhouse tournaments!
+      // hourly crazyhouse/chess960 tournaments!
       (0 to 6).toList.flatMap { hourDelta =>
         val date = rightNow plusHours hourDelta
         val hour = date.getHourOfDay
         val speed = hour % 6 match {
-          case 0 | 3 => Bullet
-          case 1 | 4 => SuperBlitz
-          case 5     => HippoBullet
+          case 1 | 4 => Bullet
+          case 2 | 5 => SuperBlitz
+          case 3     => HippoBullet
```

Comment:
why did you change these? I made them different from the ones in the atomic/antichess section on purpose, to maximize available choices (if for instance you don't want to play any bullet)

Reply:
Excellent question, I just did things the same way they were for the other 2 variant rotations. I guess if this was intended to be different, this is pointless. I also just realized that the way I coded this has hippobullet 960 tournaments instead of blitz 960 tournaments, which is almost certainly undesired.